Extensions/Content/Dnn.PersonaBar.Extensions/WebApps/Licensing.Web/src/components/platform/index.jsx

```diff
@@ -1,91 +1,101 @@
-                <div className="link-evoq-wrapper">
-                    <div className="link-evoq" onClick={this.onEvoqClick.bind(this) }>
-                        <div className="star-icon" dangerouslySetInnerHTML={{ __html: starIcon }} />
-                        <div className="link-evoq-header">{resx.get("CheckOutEvoq.Header") }</div>
-                        <div className="link-evoq-desc">{resx.get("CheckOutEvoq") }</div>
+                <div className="link-docs-wrapper" title={resx.get("Docs.Header")} onClick={this.onDocsClick.bind(this) }>
+                    <div className="link-docs">
+                        <div className="docs-icon" dangerouslySetInnerHTML={{ __html: docsIcon }} />
+                        <div className="link-docs-header">{resx.get("Docs.Header") }</div>
+                        <div className="link-docs-desc">{resx.get("Docs") }</div>
                     </div>
                 </div>
-                <div className="link-upgrade-wrapper">
-                    <div className="link-upgrade" onClick={this.onUpgradeClick.bind(this) }>
-                        <div className="evoq-icon" dangerouslySetInnerHTML={{ __html: evoqIcon }} />
-                        <div className="link-upgrade-header">{resx.get("UpgradeToEvoq.Header") }</div>
-                        <div className="link-upgrade-desc">{resx.get("UpgradeToEvoq") }</div>
+                <div className="link-community-wrapper">
+                    <div className="link-community" title={resx.get("Community.Header")} onClick={this.onCommunityClick.bind(this) }>
+                        <div className="dnn-icon" dangerouslySetInnerHTML={{ __html: dnnIcon }} />
+                        <div className="link-community-header">{resx.get("Community.Header") }</div>
+                        <div className="link-community-desc">{resx.get("Community") }</div>
                     </div>
                 </div>
-                <div className="link-doc-wrapper" onClick={this.onDocCenterClick.bind(this) }>
-                    <div className="link-doc">
-                        <div className="info-icon" dangerouslySetInnerHTML={{ __html: infoIcon }} />
-                        <div className="link-doc-header">{resx.get("DocumentCenter.Header") }</div>
-                        <div className="link-doc-desc">{resx.get("DocumentCenter") }</div>
+                <div className="link-github-wrapper">
+                    <div className="link-github" title={resx.get("GitHub.Header")} onClick={this.onGitHubClick.bind(this)}>
+                        <div className="github-icon" dangerouslySetInnerHTML={{ __html: githubIcon }} />
+                        <div className="link-github-header">{resx.get("GitHub.Header")}</div>
+                        <div className="link-github-desc">{resx.get("GitHub")}</div>
                     </div>
                 </div>
             </div>
         );
     }
 
+    UNSAFE_componentWillMount() {
```

Comment:
Can you move this into componentDidMount to avoid the soon to be deprecated method usage ?

Reply:
Yes, great suggestion - thanks @valadas - standby for a new commit.

Reply:
Okay, new commit pushed.